base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp

```diff
@@ -214,13 +222,8 @@ class CvdStartCommandHandler : public CvdServerHandler {
     MarkLockfiles(group_info, InUseState::kInUse);
   }
 
-  /*
-   * wait, remove the instance group if start failed, filling out the
-   * response.
-   */
-  Result<cvd::Response> PostStartExecutionActions(
-      selector::GroupCreationInfo& group_creation_info,
-      RequestWithStdio request);
+  void PostStartExecutionActions(
```

Comment:
should we keep the documentation for this method?

Reply:
The method didn't do that anymore